presto-product-tests/README.md

```diff
@@ -17,153 +17,231 @@
     pip install docker-compose
     ```
 
-#### On Mac OS X
+### OS X (10.8 "Mountain Lion" or newer)
+
+The Docker daemon cannot run natively on OS X because it uses Linux-specific
+kernel features. Instead, the Docker daemon runs in a Linux VM created by
+the ``docker-machine`` binary. Docker containers run in the Linux VM and are
+controlled by the ``docker`` client binary that runs natively on OS X.
+Both ``docker-machine`` and ``docker`` are included in the ``docker-toolbox``
+package, which must be installed.
 
 * [```docker-toolbox >= 1.10```](https://www.docker.com/products/docker-toolbox)
 
-On Mac OS X installing docker-toolbox gives access to a preconfigured shell environment
-with ```docker``` and ```docker-compose``` available. The shortcut to this preconfigured
-shell environment for docker may be found in Applications/Docker as "Docker Quickstart Terminal."
-Note that all commands given in later parts of these instructions should be run from this
-environment.
+In addition to ``docker-machine`` and ``docker``, the ``docker-toolbox``
+package also install ``docker-compose``, which is a multi-container
+orchestration Python utility. To gain access to these utilities, start the
+pre-configured shell environment by double-clicking on the "Docker Quickstart
+Terminal" icon located in ~/Applications/Docker. Note that all commands listed
+in subsequent parts of this tutorial must be run within such a pre-configured
+shell.
 
-##### Setting up virtual machine for docker
+#### Setting up a Linux VM for Docker
 
-As part of the ```docker-toolbox``` installation process a VirtualBox machine with the
-name ```default``` is created. In order to use this machine to run product-tests it must be
- configured with at least 4GB memory. This configuration can be done using following commands:
+The ``docker-toolbox`` installation creates a VirtualBox VM called ``default``.
+To run product-tests on the ``default`` VM, it must be re-configured to use
+4GB of memory with the following commands:
 
-```
-docker-machine stop
-vboxmanage modifyvm default --memory 4096
-docker-machine start
-```
+    ```
+    docker-machine stop
+    vboxmanage modifyvm default --memory 4096
+    docker-machine start
+    ```
 
-If one does not want to use the ```default``` machine to run product tests, the following
- steps may be used to create and use a new machine named ```<machine>```:
+Alternatively, if you do not want to use the ```default``` VM to run the
+product tests, you can create a new VM with the commands below. Note that
+the ``default`` VM will always be running when you start a new pre-configured
+shell environment. Permanently removing or replacing the ``default`` VM
+is beyond the scope of this tutorial.
 
-* To create a virtual machine with name <machine> (required one time):
+* Create a VM called <machine>. This should be done only once and not
```

Comment:
I guess you forgot to surround \<machine\> with quotes so as to make it as \`\`\`\<machine\>\`\`\`.